src/pages/users/widgets/EditUserForm.vue

```diff
@@ -0,0 +1,115 @@
+<script setup lang="ts">
```

Comment:
When you try to update user - it works first time.

But after you open modal again and try to edit something - it doesn't work.